apps/freelance/src/pages/PasswordResetRequest/index.tsx

```diff
@@ -13,20 +14,18 @@ type Inputs = {
 const PasswordResetRequest = () => {
   const { t } = useTranslation();
   const { handleSubmit, control } = useForm<Inputs>();
-  const [loading, setLoading] = useState<boolean>(false);
-  const [email, setEmail] = useState<string>('');
+  const email = useRef<string>('');
```

Comment:
why refs needed, I dont like refs, if you use it - you should have good reason

Reply:
@lyudad change it to useState